app/bankaccountverification/web/business/BusinessAccountDetailsView.scala.html

```diff
@@ -44,13 +44,15 @@ <h1 class="govuk-heading-xl">@messages("label.accountDetails.heading")</h1>
             autocomplete = Some("off"),
             formGroupClasses = "form-field-group", hint = Some(Hint(content = HtmlContent(messages("hint.sortCode")))),
             name = detailsForm("sortCode").name, id = detailsForm("sortCode").name, classes = "govuk-input--width-5",
+            pattern=Some("[0-9]"), inputmode=Some("numeric"),
```

Comment:
I have a vague memory that we needed to be able to handle sortcodes with spaces and tabs - is this no longer the case?